### PR TITLE
Add new OIDC page

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
A new page for OIDC

Table of claims/descriptions:
Screen Shot 2022-03-23 at 2 57 10 PM

# Reasons
New stuff is fun and should be documented!
[Jira ticket](https://circleci.atlassian.net/jira/software/projects/DOCSTEAM/boards/412?selectedIssue=DOCSTEAM-227)

### CODE REVIEW

**Comment by devinmarieb** [comment]
@aterweele I've included a screen shot of the claims section as I think it presents better as a table than a bullet point list. I've also included the page in the side nav in the features section of Pipelines right under Contexts. If you have a suggestion of a better place for it, please let me know.

File: jekyll/_cci2/openid-connect-tokens.adoc [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  my-workflow:
+    jobs:
+      - run-tests:
+          context:
+            - my-context
+```
+
+== Setting up your cloud service
+
+_Consult your cloud service's documentation for how to add an identity provider._
```
Comment (aterweele) [inline jekyll/_cci2/openid-connect-tokens.adoc:34]: A change from the [old PR](https://github.com/circleci/circleci-docs/pull/6578) that didn't make it over here:

> Consult your cloud service's documentation for how to add an identity provider, for example, AWS's [Creating OpenID Connect (OIDC) identity providers](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html) or Google Cloud Platform's [Configuring workload identity federation](https://cloud.google.com/iam/docs/configuring-workload-identity-federation#oidc).

Could we get that in here?

File: jekyll/_cci2/openid-connect-tokens.adoc [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+The https://openid.net/specs/openid-connect-core-1_0.html#Terminology[OpenID Provider] is unique to your organization. The URL is `\https://oidc.circleci.com/org/<organization-id>`, where `<organization-id>` is the UUID that represents your organization. 
+
+You can find the UUID by navigating to **Organization Settings > Contexts** on the https://app.circleci.com/[CircleCI web app].
```
Comment (aterweele) [inline jekyll/_cci2/openid-connect-tokens.adoc:38]: Maybe this could be a single paragraph?

**Comment by devinmarieb** [comment]
@aterweele changes are up.

**Review by kbatuigas — APPROVED** [review]
🙏

**Review by aterweele — APPROVED** [review]
Thanks @devinmarieb !

**Pull request merged**